### DISCUSSION TITLE
notepadqq on Zorin/Ubuntu stopped showing scrollbar and stopped reacting to mouse scroll

### DISCUSSION THREAD

**Discussion by kofaysi** [notepadqq/notepadqq#1125] [General]
My notepadqq v1.4.8 on Zorin 17.2 (Ubuntu) stopped showing scrolbar and stopped reacting to mouse scroll. Any ideas how to troubleshoot the issue, please?

**Comment by rsaxvc** [comment]
Linking related issue: https://github.com/notepadqq/notepadqq/issues/1134
